### ISSUE TITLE
Breaks when rbenv version is `system`

### ISSUE THREAD

**Issue by jasonkarns** [mlafeldt/rbenv-man#3]
``` sh
$ RBENV_VERSION=system rbenv man -w ruby
No manual entry for ruby
```

When it should print same output as:

``` sh
$ man -w ruby
/usr/share/man/man1/ruby.1
```

**Comment by mlafeldt** [comment]
I guess this was fixed by #1?

**Comment by jasonkarns** [comment]
fixed by #1 (and more _correctly_ fixed by #4)

It's fixed by #1 because when `$RBENV_ROOT/versions/system/share/man` prepended to `MANPATH`, then the system ruby man page isn't found under `$RBENV_ROOT/versions/system/share/man` (which doesn't even exist), then it _is_ found under the original `MANPATH`. So the bug is fixed but rather by side effect.

#4 fixes this issue more correctly because `rbenv-prefix` returns the actual location for system ruby, not just relying on the fallback support of multi-directory `MANPATH`. So `$(rbenv-prefix)/share/man` is the actual location of the system ruby manpages.
